data/hong-codes.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Hong Dang",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Transportation Planner turned Software Developing",
+  "avatar": "https://github.com/hong-codes.png",
+   "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/hong-codes",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/hongcodes",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/hongcodes/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Follow me on Twitch",
+      "url": "https://www.twitch.tv/hongcodes",
+      "icon": "FaTwitch"
+    },
```

Comment:
Looks good, but this trailing comma `,` is not needed

Reply:
Thank you, I updated it now.